source/optional_features/block_prepaid_cards/index.html.md.erb

```diff
@@ -0,0 +1,17 @@
+---
+title: Block prepaid cards
+weight: 140
+---
+
+# Block prepaid cards
+
+You can block users from paying with prepaid cards. You should only do this if users are making fraudulent payments with prepaid cards.
+
+You cannot block most Mastercard prepaid cards, because we usually cannot tell if Mastercard cards are prepaid or not.
```

Comment:
I still think this needs a more explicit explanation of what happens if you choose to block prepaid cards but someone tries to pay with a prepaid Mastercard card. Right now, it almost reads like you can’t turn the feature on if you want to block prepaid Mastercard cards.

Reply:
@alexbishop1 I chatted to Miriam and we feel this is probably ok as is, but we're going to see if our 2ier reads it the same way.